src/libcrun/cgroup.c

```diff
@@ -91,6 +91,38 @@ libcrun_get_cgroup_mode (libcrun_error_t *err)
   return stat.f_type == CGROUP2_SUPER_MAGIC ? CGROUP_MODE_HYBRID : CGROUP_MODE_LEGACY;
 }
 
+static bool
+is_rwm (bool *rwm, const char *str, libcrun_error_t *err)
```

Comment:
probably this boolean value can be returned as the return value

Reply:
ouch, forgot to change the type when I moved the boolean to an argument